src/main/java/in/zapr/druid/druidry/Context.java

```diff
@@ -49,6 +49,9 @@
     private Integer maxIntermediateRows;
     private Boolean groupByIsSingleThreaded;
 
+    private Boolean applyLimitPushDown;
```

Comment:
As per your link of [GroupByQueryConfig.java](https://github.com/apache/incubator-druid/blob/72a571fbf71d8b3209377a2d5bdd5cfac08f95a7/processing/src/main/java/org/apache/druid/query/groupby/GroupByQueryConfig.java#L31) in [incubator-druid](https://github.com/apache/incubator-druid) `applyLimitPushDown` is defined but neither assigned any value nor used to form `GroupByQueryConfig`.

Also `applyLimitPushDown` is not there is the [documentation](http://druid.io/docs/latest/querying/groupbyquery.html#groupby-v2-configurations).

I think we can omit it for now. Have you used `applyLimitPushDown` in any query ? Does it work ? What does it do ?

Reply:
please see https://github.com/apache/incubator-druid/issues/4739

we use it to turn limit pushdown off for large queries so they don't fail.